app/views/admin/components/MaintenanceModal.vue

```diff
@@ -0,0 +1,55 @@
+<script>
+  export default Vue.extend({
+    data() {
+     return {
+       hour: 1,
+     }
+    },
+    methods: {
+      submit: async function(){
+        await fetch('/admin/maintenance-time', {
```

Comment:
Should we just use `fetchJson` from [app/core/api/fetch-json.coffee](https://github.com/codecombat/codecombat/blob/master/app/core/api/fetch-json.coffee) instead?